### PR TITLE
feat(addNewIcons): kick start adding new icons

### PR DESCRIPTION
**Pull request by ChanJit**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

Job Detail Page required new icons and those icon not existing in seek style guide 

BREAKING CHANGE:

FacebookIcon and TimeIcon added new attribute

RFC URL:

none

MIGRATION GUIDE:

none

EXAMPLE USAGE:
```<FacebookIcon square={true} />```

localhost_3000_icons 3

### CODE REVIEW

**Comment by mengtzu** [comment]
Tests failing looks like a dependency version issue?  Should go away if you can guarantee you saved your snapshots with the same version circleci is using

**Comment by ChanJit** [comment]
@mengtzu yes it was due to dependency version issue, I try to use @ZhenYongSeekAsia `package-lock.json` it success. Just wonder if I update the snapshot will it affect the reset of the member.

**Comment by mengtzu** [comment]
@ChanJit we want it to affect everybody, assuming the snapshots are generated with the same dependencies CircleCI will run.  If anyone's dependencies are out of date locally they'll need to `npm install` but that's normal.

**Comment by ChanJit** [comment]
ok cool I will split the fix to different pull request

File: react/CalendarIcon/CalenderIcon.iconSketch.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import CalenderIcon from './CalenderIcon';
```
Comment (PepeFranco) [inline react/CalendarIcon/CalenderIcon.iconSketch.js]: change to ```CalendarIcon```

**Pull request merged**